scrapy/commands/genspider.py

```diff
@@ -135,6 +112,32 @@ def _list_templates(self):
             if filename.endswith('.tmpl'):
                 print("  %s" % splitext(filename)[0])
 
+    def _spider_exists(self, name):
+        if not self.settings.get('NEWSPIDER_MODULE'):
+            # if run as a standalone command and file with same filename already exists
+            if exists(name + ".py"):
+                print("%s already exists" % (abspath(name + ".py")))
+                return True
```

Comment:
:lipstick: If you add a `return False` statement after this `if`, at the same indentation level as the `if`, then the `else` below of the upper `if` is not needed anymore, allowing you to remove 1 level of indentation from all the code below.